dependency-lint.yml

```diff
@@ -17,7 +17,8 @@ executedModules:
     root: 'scripts/*.sh'
 
 ignoreErrors:
-  missing: []
+  missing:
+    - '@cucumber/cucumber' # false positive of own package, not sure why yet
```

Comment:
Looks like at some point the feature tests are creating `node_modules/@cucumber/cucumber` (its not there originally). Let me check if an analogous thing is happening on master

Reply:
Look like its the global install test

Reply:
Okay, think I understand this now. So for the global install test, we symlink all folders under node_modules into a tmp dir and we don't account for scoped packages and thus symlink the `@cucumber` folder. Then to create the "global" install we copy some cucumber source there. But due to the symlink we are actually editing the node_modules folder.

I think we can change how the modules are symlinked to avoid this.

Reply:
Follow up as to why we saw this on travis only. So dependency lint runs before feature tests and this issue arose if running dependency lint after running feature tests. Travis caches the node modules folder at the end of the build and thus the error appeared after the dependencies got cached with the copy of our source code in the node modules folder.